ReactWindows/ChakraBridge/ChakraHost.cpp

```diff
@@ -67,12 +68,38 @@ JsValueRef CALLBACK ConsoleError(JsValueRef callee, bool isConstructCall, JsValu
     return InvokeConsole(L"error", callee, isConstructCall, arguments, argumentCount, callbackState);
 }
 
-JsErrorCode ChakraHost::RunScriptFromFile(const wchar_t* szFileName, const wchar_t* szSourceUri, JsValueRef* result)
+JsErrorCode LoadByteCode(const wchar_t* szPath, BYTE** pData)
+{
+    FILE *file;
+    *pData = nullptr;
+
+    if (_wfopen_s(&file, szPath, L"rb"))
```

Comment:
I wonder whether you would want to use mapped file here, where the pages in the file will only be read when used.
So if there are pages containing function that is never executed, it will not be read.  Should have some memory saving too.

The trade off is that you pay for file I/O during execution time, versus here you pay for it upfront.